src/main/java/net/minecraftforge/client/event/OrientCameraEvent.java

```diff
@@ -0,0 +1,17 @@
+package net.minecraftforge.client.event;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.EntityPlayer;
+import cpw.mods.fml.common.eventhandler.Event;
+
+public class OrientCameraEvent extends Event
+{
+    public final float partialTicks;
+    public final EntityPlayer player;
+    
+    public OrientCameraEvent(Minecraft mc, float partialTicks)
```

Comment:
Might want to use the entity that the camera is focused on instead of the player. (For example, you can change the view entity to a sheep)